src/MountedTraversal.js

```diff
@@ -49,9 +49,8 @@ export function instHasClassName(inst, className) {
   if (!isDOMComponent(inst)) {
     return false;
   }
-  let classes = findDOMNode(inst).className || '';
-  classes = classes.replace(/\s/g, ' ');
-  return ` ${classes} `.indexOf(` ${className} `) > -1;
+  let classes = findDOMNode(inst).classList;
```

Comment:
The only advantages I can see are a) if the `className` approach *can't* be made to work, or b) in the far future when we could drop `className` support, we'd be able to cleanly delete a branch of code.